time/src/main/java/io/spine/time/Timestamps2.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package io.spine.time;
+
+import com.google.common.base.Converter;
+import com.google.protobuf.Timestamp;
+import io.spine.string.Stringifiers;
+
+import java.time.Instant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Utilities class for working with {@link com.google.protobuf.Timestamp Timestamp} values in
+ * addition to those available from {@link com.google.protobuf.util.Timestamps Timestamps} utility
+ * class from the Protobuf Util library.
+ *
+ * @deprecated Use {@link TimestampTemporal} instead. Also, consider domain-specific types for time
+ *         representation.
+ */
+@Deprecated
+public final class Timestamps2 {
+
+    /** Prevent instantiation of this utility class. */
+    private Timestamps2() {
+    }
+
+    /**
+     * Converts the passed timestamp to {@code Instant}.
+     *
+     * @deprecated Use {@link Temporal}s instead.
```

Comment:
In this case and in similar cases please refer to a particular method.

If no replacement is available yet, let's not deprecate for now.